### PR TITLE
Comments api

### PR DESCRIPTION
**Pull request by z4cope**
- Displayed Comments pop up with selected item's details.
- Displayed comments for a given item on the Comments pop-up.
- Created an add new comment feature.
- Added comments counter at the top of the page.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @z4cope @ZainabElzubair,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You are required to have merged all your changes in the dev branch, and then raise Pull Request from dev to main and submit the link to that PR instead.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: dist/index.html [hunk 1]
```diff
@@ -1,31 +1,61 @@
...
+          <li class="nav-pills">Chicken</li>
+          <li class="nav-pills">Beef</li>
+          <li class="nav-pills">Seafood</li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <!-- <div class="count">
+        <h2>Empty</h2>
+      </div> -->
```
Comment (Zainab-Alzubair) [inline dist/index.html:27]: Please remove the commented code from the file to make your code clean

**Review by Zainab-Alzubair — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @z4cope  👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  


## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Follow best practices  ✔️

## Required Changes:
- Please remove the commented code from the file to make your code clean
------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zainab-Alzubair — APPROVED** [review]
## Status: Approved ✔️
**Hi @z4cope ,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [OPTIONAL] 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**